### ISSUE TITLE
Using Google noto.otf font generates "Postscript outlines are not supported" Error

### ISSUE THREAD

**Issue by pfleu** [mpdf/mpdf#1365]
Using Mpdf v8.
I'm trying to use custom fonts to display Asian characters.
Getting error "Postscript outlines are not supported" though docs say otf format is supported.
Is there a way to use custom otf fonts to display Asian text?
Thanks !

**Comment by pfleu** [comment]
@finwe I guess you closed my issue for some good reason, but could you tell me why please?

**Comment by finwe** [comment]
Did you read the issue template before you just deleted it?

**Comment by finwe** [comment]
Did you miss the General troubleshooting category in the New Issue menu?

**Comment by finwe** [comment]
(There are OTF fonts with TrueType and PostScript outlines. the latter are not supported. Would this error message be more understandable? https://github.com/mpdf/mpdf/commit/5f0174a13fcb0e48d2183c285cb79da4e0846754)

**Comment by pfleu** [comment]
As you might have guessed I'm not used to post issue, sorry for missing out the steps you mentioned. I clicked "new issue" without being logged yet and there was no issue template nor General troubleshooting category when I got to write my issue, just an empty textarea.
Thank you anyway for having taken the time to answer me, and yes the error message you mentioned would have been more understandable.
And thanks also for your work, I like using Mpdf. Regards.
